### ISSUE TITLE
Toggle doesn't announce when it is changed

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#1619]
Environment: IE11 and JAWS 18
Are you willing to submit a PR to fix? No. I need help
Requested priority: High

Toggle does not say when it is turned "off" or "on". Specifically, it should announce "<label name><state>"

**Comment by scriby** [comment]
I was looking into this today as well and as far as I can tell things are implemented properly. The button has type="button" and sets aria-pressed. (I used https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_button_role as a reference)

I've noticed it doesn't read anything in both JAWS and VoiceOver (OSX), but it does read off/on in Narrator + Edge.

There's a fair chance this is an issue with the screenreaders, but I'm wondering if we can do anything with aria-live to announce the changes more directly.

**Comment by betrue-final-final** [comment]
I'm not sure that buttons would read what we need. Are we not sure that these shouldn't be checkboxes in ARIA? I'm not dogmatic, just want it to work properly.

________________________________

**Comment by mdahamiwal** [comment]
These were implemented as checked boxes earlier but due to visual design (which is toggle button) we implemented the aria-pressed pattern. This is a valid aria design pattern as pointed out by @scriby. As I said on another thread, a lot of ATs and browsers are still catching up on aria 1.1 guidelines and may take sometime.

**Comment by betrue-final-final** [comment]
Whatever you use is ok with me, but we need to support JAWS and IE. This combo is like Chrome for the blind world. Sure NVDA and Firefox are used, but we have to support this combo.

It's not realistic to tell our users, especially government agencies, to switch their software. This is a serious issue.

Once again here is @Anne<mailto:annetay@microsoft.com> Taylor, who can pile on. Note that she is from CELA and former technology officer for the NFB. @Clint<mailto:Clinton.Covington@microsoft.com> as well for accessibility in Office 365.

I get it, IE and JAWS are a bear. I would love nothing more than to work with the smoothest combination of software so I don't have to do this testing so much. Let me know if there is some way I can help.

________________________________

**Comment by betrue-final-final** [comment]
Sorry for the naive question... I'm not deep in the technical aspects of this issue but is this something we could have Jaws address? We have an open PO with them to fix issues that are critical to our scenarios.
